### PR TITLE
chore(release): 0.6.0

### PR DESCRIPTION
**Pull request by TarikGul**

### CODE REVIEW

**Review by insipx — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 0.6.0 August 5, 2021
```
Comment (niklasad1) [inline CHANGELOG.md:3]: I don't really get why this is a breaking change?

Comment (TarikGul) [reply CHANGELOG.md:3]: Not a breaking change, but just a feature.

Comment (niklasad1) [reply CHANGELOG.md:3]: ah right but I think it's not strictly necessary as `MAJOR` version is 0 in this case but it's fine either way.

**Comment by TarikGul** [comment]
Closing PR, changing version to v0.5.5.

**Pull request closed without merging**